### PR TITLE
clearing up the admin panel documentation

### PR DESCRIPTION
**Pull request by tobiasd**
part of see #2194 - the German translation is still missing

### CODE REVIEW

File: doc/Settings.md [hunk 1]
```diff
@@ -1,230 +1,259 @@
...
-===
-Here are some of the built-in features which don't have an exposed interface or are otherwise undocumented.
-Configuration settings are stored in the file ".htconfig.php".
-Edit this file with a text editor to make the desired changes.
-Several system settings are already documented in that file and will not be covered here. 
+# Settings
 
-Hot Keys
----
+If you are the admin of a Friendica node, you have access to the sp called **Admin Panel** where you can configure your Friendica node.
```
Comment (annando) [inline doc/Settings.md]: "s**o** called"

File: doc/Settings.md [hunk 2]
```diff
@@ -1,230 +1,259 @@
...
 
-Please be aware that turning on the logging can fill up the disk space on your server relatively quick.
-You should take preventions with e.g. [logrotate](https://en.wikipedia.org/wiki/Log_rotation) or similar tools.
+The log section of the admin panel is seperated into two pages.
+On the first, following the "log" link, you can configure how much Friendica shall log.
+And on the second you can read the log.
 
-###PHP error logging
+When you are using the default configuration for Apache, you should choose a name for the logfile ending in ``.log`` or ``.out``.
+But don't call it ``friendica.log`` (see [issue 2209](https://github.com/friendica/friendica/issues/2209)).
```
Comment (annando) [inline doc/Settings.md]: You can call it friendica.log - when you don't place it in the web root. (Which is recommened)

Comment (tobiasd) [inline doc/Settings.md]: Recommended where?

Comment (annando) [inline doc/Settings.md]: I think it is a security rule to only have stuff in your web root that really should be accessed from the web server.

Comment (tobiasd) [inline doc/Settings.md]: Then I'll write it down as a recommendation instead.

Through we have rewrite rules to prohibit access to .log files...  But we should keep them in as safeguard for people who did not know your recommendation before...

Comment (annando) [inline doc/Settings.md]: Have you already added it to the doc?

Comment (tobiasd) [inline doc/Settings.md]: yes, but I pushed the wrong branch yesterday ... pushed the correct one now

**Pull request merged**